util/image_viewer/spotfinder_frame.py

```diff
@@ -124,6 +124,7 @@ def __init__(self, *args, **kwds):
         self._mask_frame = None
 
         self.show_all_pix_timer = time_log("show_all_pix")
+        self.show_thresh_pix_timer = time_log("show_thresh_pix")
```

Comment:
I think all these `show_*_timer`s should be removed - I'm not sure their output goes anywhere, so they are just adding clutter to the code.

Reply:
If they do nothing then I agree they should be removed. I think that should be outside this PR though so it doesn't get lost in the squash merge commit message